app/views/foreman_leapp/job_templates/leapp_preupgrade.erb

```diff
@@ -6,12 +6,20 @@ description_format: 'Upgradeability check for RHEL host'
 provider_type: SSH
 feature: leapp_preupgrade
 model: JobTemplate
+template_inputs:
+- name: Channel
+  description: Set if the target OS is required to use any Extended Upgrade Support (EUS), Advanced Update Support (AUS), Update Services for SAP Solutions (E4S) repository or none.
+  input_type: user
+  required: true
+  default: "none"
+  options: "none\neus\naus\ne4s"
```

Comment:
My leapp (0.15.1) also has also these two options:
* `ga`
* `tuv`

Should they be included as well?

Reply:
reworked to be 1:1 with upstream: 
https://github.com/oamg/leapp-repository/blob/84d6ce3073e646e8740b72a5e7edda056c1b324a/commands/preupgrade/__init__.py#L28